<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flurdisplay Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>23</y>
       <width>441</width>
       <height>112</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelDevice">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFirmware">
          <property name="text">
           <string>Firmware:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDevInterface">
          <property name="text">
           <string>Device interface:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDevInterfaceParam">
          <property name="text">
           <string>Device interface param:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelHostInterface">
          <property name="text">
           <string>Host interface:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="configButton">
        <property name="text">
         <string>Set &amp;up...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Text</string>
    </property>
    <widget class="QLineEdit" name="ledDisplay">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>40</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="indicatorMultipleText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startStopButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>St&amp;art</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>195</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="testPassedButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Test &amp;passed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testFailedButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Test &amp;failed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Tone</string>
    </property>
    <widget class="QLabel" name="labelTone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>none</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Communication</string>
    </property>
    <widget class="QPlainTextEdit" name="protocolView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
